### DISCUSSION TITLE
Compile and Install Errors

### DISCUSSION THREAD

**Discussion by donkarnag** [joomengine/Joomla-Component-Builder#757] [Q&A] [answered]
I have been noticing some performance issues when Compiling.  The result is always 'Successfully Compiled!', however sometimes the package appears incomplete and fails on installation.  By incomplete, I mean that the 'Total Folders Created', 'Total Files Created', etc. are less than expected.  For my current component, 'complete' is as shown with about 30153 lines of code and as shown.  To 'resolve', I keep compiling until I see a package that is about the size I expect it to be.  

My first suspicion is that since this is on shared hosting I bump up on time limits, but I cannot confirm that - it just feels right.  And before I move to a dedicated host, I wanted to see if there are others things to check.  

Has anyone run into this?  I don't see anything on the Wiki or Discussions.

Thanks!

Running JCB version 2.12.9
PHP Built On | Linux a2plcpnl0600.prod.iad2.secureserver.net 2.6.32-954.3.5.lve1.4.82.el6.x86_64 #1 SMP Tue Mar 16 17:47:27 UTC 2021 x86_64
Database Type | mysql
Database Version | 5.6.51-cll-lve
Database Collation | latin1_swedish_ci
Database Connection Collation | utf8mb4_general_ci
PHP Version | 7.3.27
Web Server | Apache
WebServer to PHP Interface | litespeed
Joomla! Version | Joomla! 3.9.27 Stable [ Amani ] 25-May-2021 15:00 GMT
Joomla! Platform Version | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36

![image](https://user-images.githubusercontent.com/20792363/120943652-1eb63480-c6e5-11eb-84fb-a9161f287544.png)

![image](https://user-images.githubusercontent.com/20792363/120943671-3a213f80-c6e5-11eb-9cc1-95dfb420467a.png)

**Comment by Llewellynvdm** [comment, marked as answer]
There has been multiple issues opened about this in the past, which a quick search in the issues area would show. Most of them the problem was Chrome. Please only use Firefox for development, and then you should have a consistent experience.

Should the issue continue, then check your environment that the folders and file permissions are correct.

We have spend much time to make sure that we use best practice on this front. So 99% of the time this is not a JCB issue, either browser related (I know that sounds crazy -- but that is Chrome for you ;) or environment issue.
